lib/src/models.dart

```diff
@@ -405,11 +405,15 @@ class Arrow implements Shape {
   final Color color;
   final SquareId orig;
   final SquareId dest;
+  /// Width of the arrow and size of its tip will be scaled by this factor.
```

Comment:
You should add an empty doc comment line between the first and second line.

```suggestion
  /// Width of the arrow and size of its tip will be scaled by this factor.
  ///
  /// If 1.0, the width will be 1/4th of the square size.
```

Reply:
wow, thanks for this great "feature", github...